content/en/docs/concepts/extend-kubernetes/extend-cluster.md

```diff
@@ -122,7 +122,7 @@ For more about Custom Resources, see the [Custom Resources concept guide](/docs/
 
 ### Combining New APIs with Automation
 
-Often, when you add a new API, you also add a control loop that reads and/or writes the new APIs. When the combination of a Custom API and a control loop is used to manage a specific, usually stateful, application, this is called the *Operator* pattern. Custom APIs and control loops can also be used to control other resources, such as storage, policies, and so on.
+Often, when you add a new API, you also add a control loop that reads and/or writes the new APIs. When the combination of a Custom API and a control loop is used to manage a specific, usually stateful, application; this is called the [Operator](/docs/concepts/extend-kubernetes/operator/) pattern. These custom APIs and control loops can also be used to control other resources, such as storage, policies, and so on.
```

Comment:
nit: do you need, `and so on`?
nit: The combination of a Custom API and a control loop is called the [Operator](/docs/concepts/extend-kubernetes/operator/) pattern. The operator pattern is used to manage specific, usually stateful, applications.

Reply:
Reworded